src/scripts/genEventing.py

```diff
@@ -729,152 +744,44 @@ def generatePlformIndependentFiles(sClrEtwAllMan,incDir,etmDummyFile):
     if not os.path.exists(incDir):
         os.makedirs(incDir)
 
-    clrallevents   = incDir + "/clretwallmain.h"
-    clrxplatevents = incDir + "/clrxplatevents.h"
-    clreventpipewriteevents = incDir + "/clreventpipewriteevents.h"
+    clrallevents = os.path.join(incDir, "clretwallmain.h")
```

Comment:
Does this need to have a capital C or does python not care?

Reply:
No that's correct. The capitol C variable is actually a file, whereas the lowercase c variable is a filename. Nasty factoring, but I wasn't trying to mess with the style in any parts I didn't touch already

Reply:
Yuck.  Ok.